### PR TITLE
rename language abbreviations

### PR DESCRIPTION
**Pull request by tink2123**
fix https://github.com/PaddlePaddle/PaddleOCR/issues/2249

### CODE REVIEW

File: configs/rec/multi_language/generate_multi_language_configs.py [hunk 1]
```diff
@@ -19,37 +19,69 @@
...
+    'fa': 'persian',
+    'ur': 'urdu',
+    'rs': 'serbian latin',
+    'oc': 'occitan',
+    'rsc': 'serbian cyrillic',
+    'bg': 'bulgarian',
+    'uk': 'ukranian',
+    'be': 'belarusian',
+    'te': 'telugu',
+    'ka': 'kannada',
```
Comment (xmy0916) [inline configs/rec/multi_language/generate_multi_language_configs.py:38]: ka -> kn

File: configs/rec/multi_language/generate_multi_language_configs.py [hunk 2]
```diff
@@ -19,37 +19,69 @@
...
+    'chinese_cht': 'chinese tradition',
```
Comment (xmy0916) [inline configs/rec/multi_language/generate_multi_language_configs.py:39]: ch_tra

File: configs/rec/multi_language/generate_multi_language_configs.py [hunk 3]
```diff
@@ -19,37 +19,69 @@
...
+    'pt': 'portuguese',
+    'ru': 'russian',
+    'ar': 'arabic',
+    'ta': 'tamil',
+    'ug': 'uyghur',
+    'fa': 'persian',
+    'ur': 'urdu',
+    'rs': 'serbian latin',
+    'oc': 'occitan',
+    'rsc': 'serbian cyrillic',
```
Comment (xmy0916) [inline configs/rec/multi_language/generate_multi_language_configs.py:33]: rs_cyrillic

File: configs/rec/multi_language/generate_multi_language_configs.py [hunk 4]
```diff
@@ -19,37 +19,69 @@
...
+    'it': 'italian',
+    'es': 'spanish',
+    'pt': 'portuguese',
+    'ru': 'russian',
+    'ar': 'arabic',
+    'ta': 'tamil',
+    'ug': 'uyghur',
+    'fa': 'persian',
+    'ur': 'urdu',
+    'rs': 'serbian latin',
```
Comment (xmy0916) [inline configs/rec/multi_language/generate_multi_language_configs.py:31]: rs_latin

File: ppocr/data/imaug/label_ops.py [hunk 5]
```diff
@@ -94,7 +94,7 @@ def __init__(self,
                  use_space_char=False):
         support_character_type = [
             'ch', 'en', 'EN_symbol', 'french', 'german', 'japan', 'korean',
-            'EN', 'it', 'xi', 'pu', 'ru', 'ar', 'ta', 'ug', 'fa', 'ur', 'rs',
+            'EN', 'it', 'es', 'pt', 'ru', 'ar', 'ta', 'ug', 'fa', 'ur', 'rs',
             'oc', 'rsc', 'bg', 'uk', 'be', 'te', 'ka', 'chinese_cht', 'hi',
             'mr', 'ne'
         ]
```
Comment (xmy0916) [inline ppocr/data/imaug/label_ops.py:100]: kn ch_tra rs_latin rs_cyrillic

Comment (xmy0916) [reply ppocr/data/imaug/label_ops.py:100]: ppocr/postprocess/rec_postprocess.py这个文件也要改

**Review by LDOUBLEV — DISMISSED** [review]

File: ppocr/data/imaug/label_ops.py [hunk 6]
```diff
@@ -94,9 +94,9 @@ def __init__(self,
                  use_space_char=False):
         support_character_type = [
             'ch', 'en', 'EN_symbol', 'french', 'german', 'japan', 'korean',
-            'EN', 'it', 'xi', 'pu', 'ru', 'ar', 'ta', 'ug', 'fa', 'ur', 'rs',
-            'oc', 'rsc', 'bg', 'uk', 'be', 'te', 'ka', 'chinese_cht', 'hi',
-            'mr', 'ne'
+            'EN', 'it', 'es', 'pt', 'ru', 'ar', 'ta', 'ug', 'fa', 'ur',
+            'rs_latin', 'oc', 'rs_cyrillic', 'bg', 'uk', 'be', 'te', 'ka',
```
Comment (xmy0916) [inline ppocr/data/imaug/label_ops.py:98]: ka > kn

File: ppocr/postprocess/rec_postprocess.py [hunk 7]
```diff
@@ -26,9 +26,9 @@ def __init__(self,
                  use_space_char=False):
         support_character_type = [
             'ch', 'en', 'EN_symbol', 'french', 'german', 'japan', 'korean',
-            'it', 'xi', 'pu', 'ru', 'ar', 'ta', 'ug', 'fa', 'ur', 'rs', 'oc',
-            'rsc', 'bg', 'uk', 'be', 'te', 'ka', 'chinese_cht', 'hi', 'mr',
```
Comment (xmy0916) [inline ppocr/postprocess/rec_postprocess.py:30]: ka -> kn

**Review by LDOUBLEV — APPROVED** [review]
LGTM

**Pull request merged**